clang/lib/StaticAnalyzer/Checkers/BuiltinFunctionChecker.cpp

```diff
@@ -91,8 +91,29 @@ QualType getOverflowBuiltinResultType(const CallEvent &Call, CheckerContext &C,
   }
 }
 
-class BuiltinFunctionChecker : public Checker<eval::Call> {
+class BuiltinFunctionChecker
+    : public Checker<eval::Call, check::PostStmt<AttributedStmt>> {
 public:
+  void checkPostStmt(const AttributedStmt *A, CheckerContext &C) const {
```

Comment:
I am a bit confused. Do we need this in both here and in AssumeModelingChecker?

Reply:
No, I think it's just some leftover codewhile experimenting with this.
Let me remove this.